include/itkCoocurrenceTextureFeaturesImageFilter.hxx

```diff
@@ -458,26 +458,23 @@ CoocurrenceTextureFeaturesImageFilter<TInputImage, TOutputImage>
 
   itkPrintSelfObjectMacro( DigitalizedInputImage );
 
-  os << indent << "NeighborhoodRadius"
+  os << indent << "NeighborhoodRadius :"
     << static_cast< typename NumericTraits<
     NeighborhoodRadiusType >::PrintType >( m_NeighborhoodRadius ) << std::endl;
 
   itkPrintSelfObjectMacro( Offsets );
 
-  os << indent << "NumberOfBinsPerAxis" << m_NumberOfBinsPerAxis << std::endl;
-  os << indent << "Min"
+  os << indent << "NumberOfBinsPerAxis :" << m_NumberOfBinsPerAxis << std::endl;
```

Comment:
To be consistent with other ITK `PrintSelf`'s, these should be `: ` instead of ` :` (colon space instead of space colon).